src/utils.jl

```diff
@@ -1,6 +1,53 @@
 export subscript, superscript
 export record_interaction
 export rotate2D, scale, Polygon, Point2f0
+export has_value!, get_value!, set_value!
```

Comment:
Why do `has_value!` and `get_value!` end in `!`? Neither are mutating their arguments.

Reply:
(Also, these are mostly internal functions, we don't need to export them)

Reply:
You're right.